overlord/fdestate/backend/reseal.go

```diff
@@ -60,12 +62,72 @@ func MockSecbootBuildPCRProtectionProfile(f func(modelParams []*secboot.SealKeyM
 
 type StateUpdater func(role string, containerRole string, bootModes []string, models []secboot.ModelForSealing, tpmPCRProfile []byte) error
 
+type comparableModel struct {
+	BrandID   string
+	SignKeyID string
+	Model     string
+	Classic   bool
+	Grade     asserts.ModelGrade
+	Series    string
+}
```

Comment:
do we need this just because there is no Series field in `modelForSealing`? Maybe worth adding a comment explaining.